tests/test_ewmh.py

```diff
@@ -348,3 +349,44 @@ def test_ewmh_make_client_urgent_no_focus_stealing(hlwm, hc_idle, x11):
     assert demandsAttent in x11.ewmh.getWmState(winHandle, str=True)
     assert 'focus' not in hlwm.list_children('clients')
     assert 'default' == hlwm.get_attr('tags.focus.name')
+
+
+@pytest.mark.parametrize('minimized', [True, False])
+def test_minimization_announced(hlwm, x11, minimized):
+    winHandle, winid = x11.create_client()
+
+    hlwm.call(f'set_attr clients.{winid}.minimized {hlwm.bool(minimized)}')
+
+    hidden = '_NET_WM_STATE_HIDDEN'
+    assert (hidden in x11.ewmh.getWmState(winHandle, str=True)) == minimized
+
+
+def xiconifywindow(display, window, screen):
+    """Implementation of XIconifyWindow()"""
+    wm_change_state = display.get_atom('WM_CHANGE_STATE')
+    event = Xlib.protocol.event.ClientMessage(
+        window=window,
+        client_type=wm_change_state,
+        data=(32, [3, 0, 0, 0, 0]))
```

Comment:
The 3 represents the IconicState, right?

Can you maybe put a variable with a more speaking name in there, so this has a less "magic number" feeling, e.g.

iconicstate = 3
...
data=(32, [iconicstate, 0, 0, 0, 0]))

Reply:
Right, I've fixed it!

Reply:
Thanks!